cdap-proto/src/main/java/co/cask/cdap/proto/ProgramRunStatus.java

```diff
@@ -53,4 +53,14 @@ public static NodeStatus toNodeStatus(ProgramRunStatus status) {
                                                          status.name()));
     }
   }
+
+  public static ProgramStatus toProgramStatus(ProgramRunStatus status) {
+    switch(status) {
+      case STARTING:
```

Comment:
I don't believe a `SUSPENDED` or `RESUMING` run record should result in a `RUNNING` status. That would deviate from current behavior.